app_flutter/test/service/appengine_cocoon_test.dart

```diff
@@ -26,6 +26,7 @@ const String jsonGetStatsResponse = '''
           "Checklist": {
             "Key": "iamatestkey", 
             "Checklist": {
+              "Branch": "master",
               "FlutterRepositoryPath": "flutter/cocoon", 
               "CreateTimestamp": 123456789, 
               "Commit": {
```

Comment:
Nit: shall we use single quote for all of them?

Reply:
JSON doesn't send single quotes, only double quotes. This is supposed to be raw JSON.